final_task/setup.py

```diff
@@ -0,0 +1,25 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="rss_reader_Anton_Zimahorau",
+    version="2.2",
+    author="Anton Zimahorau",
+    author_email="antonzimahorau@gmail.com",
+    description="Command-line utility which receives RSS URL",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/AntonZimahorau/FinalTaskRssParser.git",
+    packages=['rss_reader', 'rss_reader.functions', 'rss_reader.classes'],
+    install_requires=['feedparser', 'lxml', 'argparse', 'requests'],
```

Comment:
Please make this correspond to your `requirements.txt` file

Reply:
Fixed